File: prak python/Shirt.py
from Clothing import Clothing

class Shirt(Clothing):

    color = ""
    sleeve_type = ""
    
    def __init__(self, color="", sleeve_type="", size="", material="", gender="", idProduct="", name="", brand="", price=""):
        super().__init__(size, material, gender)
        self.color = color
        self.sleeve_type = sleeve_type
        self.setIdProduct(idProduct)
        self.setName(name)
        self.setBrand(brand)
        self.setPrice(price)

    def getColor(self):
        return self.color

    def setColor(self, color):
        self.color = color

    def getSleeve_Type(self):
        return self.sleeve_type
    
    def setSleeve_Type(self, sleeve_type):
        self.sleeve_type = sleeve_type

File: prak c++/Shirt.cpp
#include "Clothing.cpp"
#include <string>

class Shirt : public Clothing {
private:
    std::string color;
    std::string sleeve_type;

public:
    Shirt() : Clothing() {
        this->color = "";
        this->sleeve_type = "";
    }

    Shirt(std::string color, std::string sleeve_type, std::string size, std::string material, std::string gender, std::string idProduct, std::string name, std::string brand, std::string price)
        : Clothing(size, material, gender) {
        this->color = color;
        this->sleeve_type = sleeve_type;
        setIdProduct(idProduct);
        setName(name);
        setBrand(brand);
        setPrice(price);
    }

    std::string getColor() {
        return this->color;
    }

    void setColor(std::string color) {
        this->color = color;
    }

    std::string getSleeveType() {
        return this->sleeve_type;
    }

    void setSleeveType(std::string sleeveType) {
        this->sleeve_type = sleeveType;
    }
};